### PR TITLE
update opt-in docs

### PR DESCRIPTION
**Pull request by amityadav0**

### CODE REVIEW

File: docs/guide/deployer.md [hunk 1]
```diff
@@ -46,6 +46,32 @@ await deployer.deployASA("ASA-2", {...}, { reserve: multsigaddr }); // this will
 For opting in to ASA, `deployer` supports two methods :-
 - `optInAcountToASA` to opt-in to a single account signed by secret key of sender.
 - `optInLsigToASA` to opt-in to a contract account (say escrow) where the account is represented by the logic signature address (`lsig.address()`).
+    - To opt in to ASA you can use either `Asset Index` or `name of the ASA`.
+    - Using Asset Index is useful when asset is not deployed using deployer.
```
Comment (robert-zaremba) [inline docs/guide/deployer.md:50]:
```suggestion
    To opt in to ASA you can use either `Asset Index` or `name of the ASA`. Using Asset Index is useful when asset is not deployed using deployer.
```

File: docs/guide/deployer.md [hunk 2]
```diff
@@ -46,6 +46,32 @@ await deployer.deployASA("ASA-2", {...}, { reserve: multsigaddr }); // this will
...
+```js
+   const execParam: ExecParams = {
+    type: TransactionType.OptInASA,
+    sign: SignType.LogicSignature,
+    fromAccountAddr: lsig.address(),
+    assetID: assetID,
+    payFlags: payFlags
+    lsig: lsig
+  };
+```
```
Comment (robert-zaremba) [inline docs/guide/deployer.md:74]: all things below "There is one more..." should be in the same bulletpoint.

File: docs/guide/deployer.md [hunk 3]
```diff
@@ -63,6 +89,37 @@ The main difference between deploying an ASA and SSC is that ASA takes `asset-na
 You can learn more about the flags from [Deployer API](https://scale-it.github.io/algo-builder/);
 You can learn more about Stateful Smart Contracts [here](https://developer.algorand.org/docs/features/asc1/stateful/).
 
+#### OptIn to SSC
+
+For opting in to SSC, `deployer` supports two methods :-
+- `optInAcountToSSC` to opt-in to a single account signed by secret key of sender.
+- `optInLsigToSSC` to opt-in to a contract account (say escrow) where the account is represented by the logic signature address (`lsig.address()`).
+  - To opt in to SSC you can use `Application Index`.
```
Comment (robert-zaremba) [inline docs/guide/deployer.md:97]: can you add a link or explain it further?

File: docs/guide/deployer.md [hunk 4]
```diff
@@ -63,6 +89,37 @@ The main difference between deploying an ASA and SSC is that ASA takes `asset-na
 You can learn more about the flags from [Deployer API](https://scale-it.github.io/algo-builder/);
 You can learn more about Stateful Smart Contracts [here](https://developer.algorand.org/docs/features/asc1/stateful/).
 
+#### OptIn to SSC
+
+For opting in to SSC, `deployer` supports two methods :-
```
Comment (robert-zaremba) [inline docs/guide/deployer.md:94]:
```suggestion
For opting in to SSC, `deployer` supports the following methods :-
```

File: docs/guide/deployer.md [hunk 5]
```diff
@@ -63,6 +89,37 @@ The main difference between deploying an ASA and SSC is that ASA takes `asset-na
...
+```js
+   const execParam: ExecParams = {
+    type: TransactionType.OptInSSC,
+    sign: SignType.LogicSignature,
+    fromAccountAddr: lsig.address(),
+    appID: appID,
+    payFlags: payFlags
+    lsig: lsig
+  };
+```
```
Comment (robert-zaremba) [inline docs/guide/deployer.md:121]: same as above. Maybe we can reword it, a bit? 

"Like with ASA, we can also use `executeTransaction` to opt-in a single account or contract account to SSC. Example: ..."

Comment (amityadav0) [reply docs/guide/deployer.md:120]: ok

File: docs/guide/deployer.md (hunk 3 above)
Comment (amityadav0) [reply docs/guide/deployer.md:96]: yes

File: docs/guide/deployer.md (hunk 2 above)
Comment (amityadav0) [reply docs/guide/deployer.md:73]: done

File: docs/guide/deployer.md [hunk 6]
```diff
@@ -43,9 +43,34 @@ await deployer.deployASA("ASA-2", {...}, { reserve: multsigaddr }); // this will
 
 #### OptIn to ASA
 
-For opting in to ASA, `deployer` supports two methods :-
+For opting in to ASA, `deployer` supports following methods :-
```
Comment (robert-zaremba) [inline docs/guide/deployer.md:46]:
```suggestion
For opting in to ASA, `deployer` supports following methods:
```

File: docs/guide/deployer.md [hunk 7]
```diff
@@ -63,6 +88,37 @@ The main difference between deploying an ASA and SSC is that ASA takes `asset-na
 You can learn more about the flags from [Deployer API](https://scale-it.github.io/algo-builder/);
 You can learn more about Stateful Smart Contracts [here](https://developer.algorand.org/docs/features/asc1/stateful/).
 
+#### OptIn to SSC
+
+For opting in to SSC, `deployer` supports the following methods :-
```
Comment (robert-zaremba) [inline docs/guide/deployer.md:93]:
```suggestion
For opting in to SSC, `deployer` supports the following methods:
```

**Review by robert-zaremba — APPROVED** [review]

**Pull request merged**